setup.py

```diff
@@ -26,64 +27,84 @@ def version():
                     line.startswith("Version: ")]
   return ceed_version[0]
 
-description = """
-libCEED: the Code for Efficient Extensible Discretization API Library
-=====================================================================
+def requirements():
+    with open('requirements.txt') as f:
+        return f.readlines()
+
+class libceed_build_ext(build_ext):
+    def run(self):
+        self.make_libceed_so()
+        build_ext.run(self)
+
+    def make_libceed_so(self):
+        import subprocess
+        subprocess.check_call(['make', '-j', 'MARCHFLAGS=-march=generic', '-B'])
```

Comment:
Maybe allow users to pass `MARCHFLAGS` via environment variable? Then you set it to `-march=generic` for the docker wheel build. Installs from source can then take advantage of optimization.

Reply:
Thanks; updated.